crates/kitsune_p2p/kitsune_p2p/src/test_util/harness_agent.rs

```diff
@@ -160,6 +160,22 @@ impl KitsuneP2pEventHandler for AgentHarness {
         Ok(async move { Ok(out) }.boxed().into())
     }
 
+    fn handle_query_agent_info_signed_near_basis(
+        &mut self,
+        _space: Arc<KitsuneSpace>,
+        _basis_loc: u32,
+        limit: u32,
+    ) -> KitsuneP2pEventHandlerResult<Vec<crate::types::agent_store::AgentInfoSigned>> {
+        // TODO - sort these?
```

Comment:
Should this be done now? It's pretty easy to sort a vec and I don't think we'll ever have enough peers for sorting the vec to be too slow.

Reply:
This is the deprecated test harness, I don't want to spend any time maintaining it...